tests/unit/test_ncf_rudder.py

```diff
@@ -125,6 +138,11 @@ def test_rudder_reporting_with_variable_content(self):
 
     generated_result = ncf_rudder.generate_rudder_reporting(self.reporting_with_var_metadata)
 
+    diff = self._unidiff_output(result, generated_result)
```

Comment:
Maybe you can add a comment so we knwo why do that (failing test output are impossible to understand otherwise)